src/it/hosting-maven_release_plugin_version-ok/invoker.properties

```diff
@@ -0,0 +1,21 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
*Licensed to the Apache Software Foundation (ASF)* -- double check where you're copying your license text from.